orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesSupplier.java

```diff
@@ -35,7 +35,7 @@
   private static final String TITUS = "titus";
   private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
   private static final String ROLLING_PUSH = "rollingpush";
-  private static final List<String> SUPPORTED_STAGES = Arrays.asList("enableServerGroup", "createServerGroup", "cloneServerGroup");
+  private static final List<String> SUPPORTED_STAGES = Arrays.asList("enableServerGroup", "createServerGroup", "cloneServerGroup", "rollingPush");
```

Comment:
why do we need create and clone if rollingpush and enable seem to be the only ones involved in migration?